in-toto-spec.md

```diff
@@ -760,6 +769,12 @@ match any materials.
   other words, this path must appear as a material and a product within this
 step and their hashes must not match.
 
+The artifact rules contained in the `"expected_materials"` and
+`"expected_products"` fields operate in a similar fashion as a firewal. This
```

Comment:
typo 'firewal'

Reply:
Thanks for the heads up. I ran a spell checker on the patch to correct this and other typos.